### ISSUE TITLE
[FR] Widgets

### ISSUE THREAD

**Issue by radical4ever** [FoedusProgramme/Gramophone#242]
Hi, so my FR is to add a few widgets that you can put on your main screen, some ideas might be the YouTube music one, the new Spotify one, etc, it's a really nice feature to have.

**Additional context**
I leave some images of widgets for context
![Screenshot_20240814-095208](https://github.com/user-attachments/assets/0e3e0df0-e479-4db4-835c-a546ab801fe7)
![Amayy_0-1689082497746~2](https://github.com/user-attachments/assets/bd07ca4c-a5f5-4173-8bf8-f7cf1aca2431)

**Comment by nift4** [comment]
google's guidelines for widgets: https://developer.android.com/docs/quality-guidelines/widget-quality#differentiated

**Comment by DCaedus** [comment]
Yes please, a widget would be great. With a favorite button if possible 🙏 
Here are some ideas

![Image](https://github.com/user-attachments/assets/ca631eea-9e8e-43b4-bb86-93f951784b1d)

**Comment by nift4** [comment]
![Image](https://github.com/user-attachments/assets/8ee89269-4008-4342-84b8-6c60a15cc815)
